packages/projection-typeorm/test/tsconfig.json

```diff
@@ -3,9 +3,7 @@
   "compilerOptions": {
     "baseUrl": "."
   },
-  "include": [
-    "../../golden-test-generator/src"
-  ],
+  "include": [".", "../../golden-test-generator/src"],
```

Comment:
Please extract this to a separate commit and apply the same change to tsconfig in other packages where applicable